core/domain/classifier/LDAStringClassifierTest.py

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from core.domain.classifier.lda_string_classifier import LDAStringClassifier
+from core.domain.classifier.LDAStringClassifier import LDAStringClassifier
```

Comment:
Nope -- from core.domain.classifier import modulename. We always import entire files (and reference a class by modulename.ClassName); we don't import specific classes.